components/markdown/src/markdown.rs

```diff
@@ -239,6 +241,158 @@ fn get_heading_refs(events: &[Event]) -> Vec<HeadingRef> {
     heading_refs
 }
 
+fn fix_github_style_footnotes(old_events: &mut Vec<Event>) {
```

Comment:
What do we need to fix?

Reply:
Renamed to `convert_footnotes_to_github_style()`.